Description: Disable intersphinx and use oslosphinx instead of oslo.sphinx
 oslo.sphinx was renamed oslosphinx...
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2014-03-21

--- python-hacking-0.8.0.orig/test-requirements.txt
+++ python-hacking-0.8.0/test-requirements.txt
@@ -3,7 +3,7 @@ discover
 fixtures>=0.3.14
 python-subunit
 sphinx>=1.1.2
-oslo.sphinx
+oslosphinx
 testrepository>=0.0.17
 testscenarios>=0.4
 testtools>=0.9.32
--- python-hacking-0.8.0.orig/doc/source/conf.py
+++ python-hacking-0.8.0/doc/source/conf.py
@@ -8,7 +8,7 @@ sys.path.insert(0, os.path.abspath('../.
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.intersphinx', 'oslo.sphinx']
+extensions = ['sphinx.ext.autodoc', 'oslosphinx']
 
 # autodoc generation is a bit aggressive and a nuisance when doing heavy
 # text edit cycles.
